scaffolding/src/main.rs

```diff
@@ -70,7 +76,25 @@ fn main() {
     });
 
     // Set up logical terminal and renderer.
-    let terminal = Rc::new(RefCell::new(Terminal::new(COLS, ROWS, tty_w)));
+    let mut config = Config::default();
+    let user_config_path = env::home_dir()
+        .unwrap()
+        .join(".scaffolding.rc");
+
+    // For now we don't care why the config failed to update, but in the
+    // reasonably near future we should figure out a more graceful way to
+    // detect and report configuration errors.
+    match config.update_from_file(&user_config_path
```

Comment:
Per the note in the config file, you can just remove all the methods on this once you have a generic signature on these config methods.